content/en/user-guide/aws/ecs/index.md

```diff
@@ -229,3 +231,25 @@ ecs_client.register_task_definition(
     volumes=[{"host": {"sourcePath": "/host/path"}, "name": "test-volume"}],
 )
 ```
+
+### CDK example
+
+The same functionality can be achieved in CDK following this example:
+
+```python
+task_definition = ecs.TaskDefinition(
+        ...
+        volumes=[
+                ecs.Volume(name="test-volume", host=ecs.Host(source_path="/host/path"))
+        ]
+)
+
+container = task_def.add_container(...)
+
+container.add_mount_points(
+        ecs.MountPoint(
+                container_path="/container/path",
+                source_volume="test-volume",
+        ),
+)
+```
```

Comment:
Are you mad, do you use 8 space indenting?!?!?! 😂 

```suggestion
```python
task_definition = ecs.TaskDefinition(
    ...
    volumes=[
        ecs.Volume(name="test-volume", host=ecs.Host(source_path="/host/path"))
    ]
)

container = task_def.add_container(...)

container.add_mount_points(
    ecs.MountPoint(
        container_path="/container/path",
        source_volume="test-volume",
    ),
)
```
```

Reply:
Not entirely sure how I ended up with that spacing 😂